### ISSUE TITLE
[BUG] allowCustom for dropdown broken

### ISSUE THREAD

**Issue by MeestorX** [bitfocus/companion#2215]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In 2.4, if you are using the `allowCustom` option on a dropdown, it no longer works. dropdowns now show a greyed out "input is not a valid value" instead of allowing a custom value to be typed in.

### Steps To Reproduce

Create an action that uses a dropdown.
Add the `.allowCustom = true` option

### Expected Behavior

You should be able to type in any value into the dropdown, but now it shows a greyed-out "input is not a valid value" and does not accept what you have typed.

### Environment (please complete the following information)

```markdown
- OS: MacOS 11.7.1
- Browser:Safari 16.1
- Companion Version: 2.4.0+4880-2cb1a768
```

**Comment by Julusian** [comment]
fixed 32f4992b786addc5bea75203e571fbc9f2d0db76

**Comment by MeestorX** [comment]
How do I tell when a change or fix (like this)has been merged?
